<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeCreate</class>
 <widget class="QDialog" name="EmployeeCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>114</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbId">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbSurname">
      <property name="text">
       <string>Surname:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbName">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbBirth">
      <property name="text">
       <string>Data of birth:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbPhoneNum">
      <property name="text">
       <string>Number of phone:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbEmail">
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbPosition">
      <property name="text">
       <string>Position:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbSalary">
      <property name="text">
       <string>Current salary:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbEmployment">
      <property name="text">
       <string>Data of employment:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="leId"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leSurname"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leBirth"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lePhoneNum"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leEmail"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lePosition"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leSalary"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leEmployment"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbCreate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Create object</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
